Mathlib/Order/Hom/Basic.lean

```diff
@@ -344,6 +344,9 @@ theorem comp_mono ⦃g₁ g₂ : β →o γ⦄ (hg : g₁ ≤ g₂) ⦃f₁ f₂
     g₁.comp f₁ ≤ g₂.comp f₂ := fun _ => (hg _).trans (g₂.mono <| hf _)
 #align order_hom.comp_mono OrderHom.comp_mono
 
+@[simp] lemma mk_comp_mk (g : β → γ) (f : α → β) (hg hf) :
```

Comment:
```suggestion
@[simp] lemma mk_comp_mk {g : β → γ} {f : α → β} (hg hf) :
```

Reply:
Not sure I agree. `f` AND `hf` are explicit arguments to `mk`. Further, we expect `f` to be short and `hf` to be long and `hf` is inferrable from the rewrite (assuming we rewrite forward), so it's easier to specify `f` than `hf`.